pkgs/servers/tautulli/default.nix

```diff
@@ -17,7 +17,10 @@ buildPythonApplication rec {
 
   installPhase = ''
     mkdir -p $out/bin $out/libexec/tautulli
-    cp -R contrib data lib plexpy Tautulli.py $out/libexec/tautulli
+    cp -R contrib data lib plexpy Tautulli.py CHANGELOG.md $out/libexec/tautulli
+
+    echo "master" > $out/libexec/tautulli/branch.txt
+    echo "v${version}" > $out/libexec/tautulli/version.txt
```

Comment:
Is this useful for the functioning of this software?

Reply:
Yes, this fixes the bug where the update prompt can't detect the correct version and shows something like `unknown version`.